maas-schemas/schemas/maas-backend/subscriptions/subscriptionOption.json

```diff
@@ -23,6 +23,18 @@
         "regionDefault": {
           "description": "Is this subscription option the default one for the region",
           "type": "boolean"
+        },
+        "personalDataCreateAllow": {
```

Comment:
What is this for? Required personal data items for this subscription option?

Reply:
yes